Anlab.Core/Models/OrderDetails.cs

```diff
@@ -29,6 +29,8 @@ public class OrderDetails
             }
         }
         public string ClientId { get; set; }
+        public decimal InternalProcessingFee { get; set; }
+        public decimal ExternalProcessingFee { get; set; }

```

Comment:
@laholstege I'd rather you dump this and just have one processingFee property.  It'll take a small amount of extra work up front, but just a little bit.  That part in react where you say `const processingFee = this.state.payment.clientType === 'uc' ? this.state.internalProcessingFee : this.state.externalProcessingFee; ` you'll have to change to do the math.Ceil(this.state.processingFee) logic, and of course you'll have to pass the appsettings.NonUcRate into the react view in order to do that.  But then every other place in here where you have two ProcessingFee props can go away